libcontainer/cgroups/systemd/apply_systemd.go

```diff
@@ -151,8 +151,8 @@ func (m *Manager) Apply(pid int) error {
 		properties []systemdDbus.Property
 	)
 
-	if c.Slice != "" {
-		slice = c.Slice
+	if c.Parent != "" {
```

Comment:
When invoked via docker, c.Parent will never be NULL. The default parent cgroup is "docker".